lib/spack/env/cc

```diff
@@ -279,6 +279,24 @@ while [ -n "$1" ]; do
     rp=""
 
     case "$1" in
+        -iquote*)
+            arg="${1#-iquote}"
+            if [ -z "$arg" ]; then shift; arg="$1"; fi
+            if system_dir "$arg"; then
+                system_includes+=("$arg")
+            else
+                includes+=("$arg")
+            fi
+            ;;
+        -isystem*)
+            arg="${1#-isystem}"
+            if [ -z "$arg" ]; then shift; arg="$1"; fi
+            if system_dir "$arg"; then
+                system_includes+=("$arg")
+            else
+                includes+=("$arg")
+            fi
+            ;;
```

Comment:
I think this is actually worth merging. @tgamblin and I had a similar conversation with the Bazel developers in https://github.com/bazelbuild/bazel/issues/10437. Can you create a separate PR just for this? @scheibelp and @becker33 may also be interested in this change.

Reply:
will do

Reply:
This has been merged